README.md

```diff
@@ -42,7 +42,7 @@ maxWidth | Image max width (ratio is preserved)
 maxHeight | Image max height (ratio is preserved)
 compressFormat | Can be either JPEG, PNG or WEBP (android only).
 quality | A number between 0 and 100. Used for the JPEG compression.
-rotation | Rotation to apply to the image, in degrees, for android only. On iOS, the resizing is done such that the orientation is always up.
+rotation | Rotation to apply to the image, in degrees, for android only. On iOS, the resizing is done such that rotation must be 0, 90, 180, or 270 degrees.
```

Comment:
Maybe we can tell that on iOS, the rotation must be a multiple of 90 degrees? :) (Because your code properly handle these cases if I'm not mistaken)